link_shortening/main.go

```diff
@@ -36,8 +42,8 @@ func shortHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if len(req.URL) < 8 {
-		http.Error(w, "Invalid request", http.StatusBadRequest)
+	if isValidURL(req.URL) != true {
```

Comment:
```suggestion
	if !isValidURL(req.URL) {
```
Возвращается же bool, его можно в if сразу использовать, а не сравнить с bool

Reply:
А в чем разница, я не очень понимаю

Reply:
Получается проще. Сам блок if работает с boolean значениями, то есть если твоя функция возвращает bool или переменная имеет тип bool, то можно сразу с bool работать и использовать оператор `!` для отрицания. Не `IsTrue == false`, а сразу `!IsTrue`.